docs/resources/cdn_domain.md

```diff
@@ -400,14 +410,51 @@ The `url_signing` block support:
   **type_c1**: Method C1.
   **type_c2**: Method C2.
 
-* `key` - (Optional, String) Specifies the authentication key contains `6` to `32` characters, including letters and digits.
+* `sign_method` - (Optional, String) Specifies the encryption algorithm type for URL authentication.
+  The default value is **md5**. The valid values are as following:
+  + **md5**
+  + **sha256**
+
+* `match_type` - (Optional, String) Specifies the authentication scope. The default value is **all**.
+  Currently, only support value is **all**, indicates that all files are involved in authentication.
+
+* `sign_arg` - (Optional, String) Specifies the authentication parameters. The default value is **auth_key**.
+  The valid length is limited from `1` to `100` characters, only letters, digits, and underscores (_) are allowed.
+  The value can not start with a digit.
+
+* `inherit_config` - (Optional, List) Specifies the details of the authentication inheritance.
+  The [inherit_config](#inherit_config_object) structure is documented below.
+
```

Comment:
-> Add authentication parameters to TS and MP4 files under M3U8/MPD index files, so that the files can be played 
  after authentication succeeds.